libr/core/cmd_anal.c

```diff
@@ -654,6 +654,7 @@ static const char *help_msg_ar[] = {
 	"ar?", " <reg>", "Show register value",
 	"arb", " <type>", "Display hexdump of the given arena",
 	"arc", " <name>", "Conditional flag registers",
+	"arcc", "", "Show calling convention defined from the register profile",
```

Comment:
I think it should be very explicit about the fact that the cc is just generically derived. So maybe:
```suggestion
	"arcc", "", "Show (fallback) calling convention automatically derived from the register profile",
```